docs/sql-reference/commands/show-tables.md

```diff
@@ -32,4 +32,4 @@ The returned table has the following columns.
 | size                        | DOUBLE      | The compressed size of the table. |
 | size_uncompressed           | DOUBLE      | The uncompressed size of the table. |
 | compression_ratio           | DOUBLE      | The compression ratio (`<size_uncompressed>`/`<size>`). |
-| number_of_segments          | INT         | The number of segments comprising the table. |
+| number_of_tablets           | INT         | The number of tablest comprising the table. |
```

Comment:
Typo fixed in release draft here: https://laughing-adventure-r4eyrz4.pages.github.io/sql-reference/commands/show-tables.html

In future, PRs for features that are not yet released should go to the contributor repo, rather than the production site, as this public site repo is public, so customers could potentially see new features here before they are released. See instructions here: https://app.getguru.com/card/TLg6eo9c/How-to-draft-edit-or-update-public-documentation

Reply:
I did follow the guidelines , and IMO it falls under "Quick fix" therefore implemented according to the flow described here:
https://app.getguru.com/card/c9A4kaEi/How-to-make-a-quick-fix-change-to-Firebolt-documentation-

Reply:
Typo fixed